### PR TITLE
Rectify typographical inaccuracies.

### PR DESCRIPTION
**Pull request by nnsW3**
This PR addresses several typographical errors across various files in the project. The changes improve readability and maintain the professional standard of the documentation and code comments.

**Justification**
Typographical errors, while minor, can detract from the overall quality of the project. Correcting these errors ensures clarity and professionalism, making the project more accessible and understandable for current and future contributors.

### CODE REVIEW

File: docs/pages/specs/zkvm-overview.mdx [hunk 1]
```diff
@@ -25,7 +25,7 @@ The first iteration of the Nexus Virtual Machine provides support for circuits e
 
 [[GGPR13](https://eprint.iacr.org/2012/215.pdf)] Rosario Gennaro, Craig Gentry, Bryan Parno, and Mariana Raykova. “Quadratic span programs and succinct NIZKs without PCPs”. In EUROCRYPT 2013.
 
-[[GWC19](https://eprint.iacr.org/2019/953)] Ariel Gabizon, Zachary J Williamson, and Oana Ciobotaru. “Plonk: Permutations over lagrange-bases for oecumenical noninteractive arguments of knowledge”. In: Cryptology ePrint Archive (2019)
+[[GWC19](https://eprint.iacr.org/2019/953)] Ariel Gabizon, Zachary J Williamson, and Oana Ciobotaru. “Plonk: Permutations over lagrange-bases for oecumenical non-interactive arguments of knowledge”. In: Cryptology ePrint Archive (2019)
```
Comment (mx00s) [inline docs/pages/specs/zkvm-overview.mdx:28]: We should stick with the actual title of the paper. It uses "noninteractive" without a hyphen.

The BibTeX on the linked page indicates the title should be as follows:

```suggestion
[[GWC19](https://eprint.iacr.org/2019/953)] Ariel Gabizon, Zachary J Williamson, and Oana Ciobotaru. “PLONK: Permutations over Lagrange-bases for Oecumenical Noninteractive arguments of Knowledge”. In: Cryptology ePrint Archive (2019)
```

**Review by mx00s — CHANGES_REQUESTED** [review]
Thanks for the PR.

I offered a small suggestion, but otherwise LGTM.

**Comment by nnsW3** [comment]
good sugestion thanks

**Comment by nnsW3** [comment]
already made the changes? could you merge?

**Pull request closed without merging**